src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -241,9 +243,17 @@ private function updateGetOperation(bool $v3, \ArrayObject $pathOperation, array
         $serializerContext = $this->getSerializerContext($operationType, false, $resourceMetadata, $operationName);
 
         $responseDefinitionKey = false;
+        $responseDefinitionKeys = [];
         $outputMetadata = $resourceMetadata->getTypedOperationAttribute($operationType, $operationName, 'output', ['class' => $resourceClass], true);
         if (null !== $outputClass = $outputMetadata['class'] ?? null) {
-            $responseDefinitionKey = $this->getDefinition($v3, $definitions, $resourceMetadata, $resourceClass, $outputClass, $serializerContext);
+            if ($v3) {
+                foreach ($mimeTypes as $mimeType) {
+                    $responseDefinitionKeys[$mimeType] = $this->getDefinition($v3, $definitions, $resourceMetadata, $resourceClass,
```

Comment:
What about:

* Renaming `SchemaFormatterInterface` in `DefinititionNormalizerInterface`
* Adding only a simple single method to this interface such as `SchemaFormatterInterface::getDefinition(string $resourceClass, string $operationType, string $operationName = null, string $inputOrOutput, array $serializerContext = [], string $mimeType): array` (other arguments such as outputClass and  can be generated using the factory services)

It would simplify the code a lot

Reply:
this has been done